rubicon/objc/objc.py

```diff
@@ -17,16 +18,34 @@
 
 ######################################################################
 
-def _find_or_error(name):
+_lib_path = ["/usr/lib"]
+_framework_path = ["/System/Library/Frameworks"]
+def _load_or_error(name):
     path = util.find_library(name)
-    if path is None:
-        raise ValueError("Library {!r} not found".format(name))
-    else:
-        return path
+    if path is not None:
+        return CDLL(path)
+
+    # On iOS (and probably also watchOS and tvOS), ctypes.util.find_library doesn't work and always returns None.
+    # This is because the sandbox hides all system libraries from the filesystem and pretends they don't exist.
+    # However they can still be loaded if the path is known, so we try to load the library from a few known locations.
```

Comment:
This must be a recent (iOS 10?) change, because it definitely worked on older versions. Is this something we should be tackling at the ctypes level itself?